packages/sanity/src/desk/components/paneHeaderActions/PaneHeaderCreateButton.tsx

```diff
@@ -87,6 +87,8 @@ export function PaneHeaderCreateButton({templateItems}: PaneHeaderCreateButtonPr
           mode="bleed"
           disabled
           data-testid="action-intent-button"
+          // This button handles the tooltip in a special way, won't reuse the forced tooltip.
```

Comment:
This buttons couldn't be refactored to the new tooltip, as they have a special behaviour for the tooltip content in https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/core/components/InsufficientPermissionsMessage.tsx